backend/src/router_test.go

```diff
@@ -515,6 +515,30 @@ func (suite *RouterSuite) TestPatchProfile() {
 	}))
 }
 
+func (suite *RouterSuite) TestPutTutorsUsername() {
+	suite.Run("Success", manualSetupTest(func() {
+		hash, _ := bcrypt.GenerateFromPassword([]byte("Password"), bcrypt.DefaultCost)
+		DB.Create(&User{Username: "Username", Password: string(hash)})
+
+		w := POST("/signin", gin.H{
+			"username": "Username",
+			"password": "Password",
+		})
+		suite.Equal(200, w.Code)
+		cookies := w.Result().Cookies()
+		suite.Equal("jwt", cookies[0].Name)
+
+		w = PUT("/tutors/Username", cookies...)
+		suite.Equal(200, w.Code)
```

Comment:
We need to assert that the user is now a tutor. Best way is probably to `GET /tutors/Username` and check the result.

Reply:
Added